engine/lite/components/kafka/src/main/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaComponentProvider.scala

```diff
@@ -67,15 +68,19 @@ class LiteKafkaComponentProvider(schemaRegistryClientFactory: ConfluentSchemaReg
     // TODO: change link to the documentation when json schema handling will be available
     val universalKafkaComponents = List(
       ComponentDefinition(KafkaUniversalName, new UniversalKafkaSourceFactory(schemaRegistryClientFactory, universalSerdeProvider, dependencies, new LiteKafkaSourceImplFactory)).withRelativeDocs(avro),
-      ComponentDefinition(KafkaUniversalName, new UniversalKafkaSinkFactory(schemaRegistryClientFactory, universalSerdeProvider, dependencies, LiteKafkaUniversalSinkImplFactory)).withRelativeDocs(avro))
-
-    lowLevelKafkaComponents ::: universalKafkaComponents
+      ComponentDefinition(KafkaUniversalName, new UniversalKafkaSinkFactory(schemaRegistryClientFactory, universalSerdeProvider, dependencies, LiteKafkaUniversalSinkImplFactory)).withRelativeDocs(avro)
+    )
+
+    //TODO: for now we add this feature flag inside kafka, when this provider can handle multiple kafka brokers move to provider config
+    val addLowLevelComponents = dependencies.config.getAs[Boolean]("kafka.addLowLevelComponents").getOrElse(true)
```

Comment:
Can you extract this flag to some constant available for both lite and flink?

Reply:
2. Can we change this flag to more declarative form like `lowLevelComponentsEnabled`? We mostly trying to avoid verbs in name of configurations flags...

Reply:
changed